### ISSUE TITLE
Allow app icons URLs to be absolute URLs

### ISSUE THREAD

**Issue by bnjbvr** [cozy-labs/cozy-registry#19]
As discussed with @nono on IRC, it seems to me that it is very strange that application's assets may be split out in several repositories: the actual icon file is in cozy-home, the icon metadata is in cozy-registry. Could we allow app icons URLs to be absolute URLs, so that I can have the control over my icons, more easily and independently of the vendor (that is, the cozy registry here), please?

**Comment by frankrousseau** [comment]
Currently, we don't want to let the choice of icons for application listed in the registry. It's because we want to keep the UI unified. We think it provides a better user experience for newcoming users. 

If you provide us an icon for an app listed in the registry, we would be glad to include it into the registry. But we require that it matches our guidelines : 
- colors must be black and transparent.
- format must be SVG.

**Comment by clochix** [comment]
I understand the will to keep control and an unified UI. But it has some drawbacks:
- some developers have spent time designing or choosing an icon for their application. Enforce them to use another one to get included inside the marketplace may prevent them from asking for inclusion;
- if we port popular applications to Cozy, it would be easier for users to recognize them if we keep the original logo.

So I think we should reconsider our politic about icons.

**Comment by bnjbvr** [comment]
Agreed with @clochix. It would be too bad that a particular Cozy application can't make it to the marketplace just because its icon doesn't fit your requirements. I'm not saying "vendor lockin" (note, it's Friday Trollday), but that would be really sad that Cozy jealously keeps control over its market because of this.

That being said, I agree that some things should not make it to the market (fake apps stealing personal data, etc.). But having a bad icon is a lesser evil.

**Comment by clochix** [comment]
@bnjbvr remind me of a long but soon dead troll on the non-free logo of some browser ;-)

**Comment by frankrousseau** [comment]
Sorry we can't do everything perfectly. I cannot say anything else that currently, the market place applications require an approved icon. 
We are overwhelmed by many other feature requests. So we won't promise anything about that subject. It requires time and attention because it breaks the current UI. Moreover users complains about the lack of continuity between apps. Adding more chaos won't improve the situation.
